sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -311,6 +347,7 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       .innerJoin("offerings.institutionLocation", "institutionLocation")
       .innerJoin("institutionLocation.institution", "institution")
       .leftJoin("offerings.assessedBy", "assessedBy")
+      .leftJoin("offerings.parentOffering", "parentOffering")
```

Comment:
Should we use `innerJoin`? @guru-aot @dheepak-aot @lewischen-aot Thoughts?

Reply:
yup, I would like to know dev's thoughts on this, that's why I added it in the PR description and dev chat

Reply:
I would agree with @andrepestana-aot on inner join as we do not expect any offering without parent offering id(with the migration we added)

@ann-aot I saw the reason in the PR description, IMO I would like to see the API fail when there is no parent_id to an offering as it could be manipulated in DB.

Reply:
updated to `leftJoin` and made `parentofferingid` mandatory in interfaced and dto

Reply:
Looks like it broke some test.

Reply:
i believe we should go with the innerjoin @ann-aot, i know the current implementation. BUt the part you made the leftJoin and made parentOfferingId mandatory solves the problem without failing the API.